ERROR	2023-02-21 19:33:34,767	[test_pytest.py:test_case1:10]	****************************************************************************************************
